app/api/shared.js

```diff
@@ -10,6 +10,10 @@ const buildPostRequest = (baseUrl) => async (endpoint, data, user) => {
 
   const { payload } = await wreck.post(`${baseUrl}/${endpoint}`, options)
 
+  if (!payload.toString().length > 0) {
```

Comment:
Not quite sure what you're trying to achieve here. Is this a test for string length === 0 ?